.github/workflows/ci.yml

```
@@ -42,17 +42,17 @@ jobs:
 
     runs-on: ${{ matrix.platform.os }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
 
       - name: Cache cargo folder
         uses: actions/cache@v1
         with:
           path: ~/.cargo
           key: ${{ matrix.platform.target }}-cargo-${{ matrix.rust_version }}
 
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@master
```

The recommended way is it to use `@stable`.

```suggestion
      - uses: dtolnay/rust-toolchain@stable
```

---

we are using an explicit toolchain here not `stable`, here is what dtolnay recommends in the action documentation:
> When passing an explicit toolchain as an input instead of @rev, you'll want to use "dtolnay/rust-toolchain@master" as the revision of the action.

---

actually looking further up, we are testing using stable only, so I will change the matrix and update this usage as well

---

https://github.com/tauri-apps/tao/pull/957